packages/server/src/i18n/utils/Translator.js

```diff
@@ -0,0 +1,23 @@
+import i18n from 'i18n';
+
+export default class Translator {
+  static translate(cookie, moduleName, messageKey) {
+    const langInCookie = cookie.match(/lang=(.*)-/);
```

Comment:
Avoid manual cookie parsing, when we are relying on the package to do that already - `universal-cookie-express`. Manual parsing is not reliable compared to using package just for this purpose.

Reply:
Fixed.